src/digid_eherkenning_oidc_generics/backends.py

```diff
@@ -68,6 +67,7 @@ class OIDCAuthenticationDigiDBackend(SoloConfigDigiDMixin, OIDCAuthenticationBac
 
     login_type = LoginTypeChoices.digid
     callback_path = reverse_lazy("digid_oidc:callback")
+    unique_id_user_fieldname = "bsn"
```

Comment:
@alextreme I figured since these names aren't gonna change, I'd keep them fixed and allow the `identifier_claim_name` to be configurable, that way `identifier_claim_name` (e.g. `sub`) will always be mapped to `unique_id_user_fieldname` (`bsn`)

Reply:
Sure makes sense